ansible/roles/filebeat/templates/ilm_policy.json.j2

```diff
@@ -0,0 +1,35 @@
+{
+  "policy": {
+    "phases": {
+      "hot": {
+        "min_age": "0ms",
+        "actions": {
+          "rollover": {
+            "max_size": "50gb",
+            "max_age": "7d"
+          }
+        }
+      },
+      "warm": {
+        "actions": {
+          "forcemerge": {
+            "max_num_segments": 1,
+            "index_codec": "best_compression"
+          },
+          "migrate": {
+            "enabled": false
+          }
+        }
+      },
+      "cold": {
+        "min_age": "14d",
+        "actions": {
+          "freeze": {},
```

Comment:
We don't need to pass anything here like `"enabled": true`?

Reply:
No, it doesn't seem like it. `auto` is the default if `setup.ilm.enabled` is not defined, and the ILM has been successfully created and enabled already using this config.

Reply:
What `auto` means? Will it freeze indices on cold phaze?

Reply:
Yes, freeze is configured. You can check the config created by this policy file in Kibana under Stack Management > Index Lifecycle Policies > Filebeat.

But freezing will not reduce the size on disk, it only reduces RAM use...